spec/features/users/show_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+describe "User Profile Page", type: :feature do
+  let(:password) { 'password' }
+  let(:user) { FactoryBot.create(:user, password: password) }
+  let(:user2) { FactoryBot.create(:user, password: password, full_name: "#{user.full_name}_other") }
+
+  it "lists the user's full name" do
+    sign_in user
+    visit profiles_me_path
+
+    expect(page).to have_text(user.full_name)
+  end
+
+  it "lists the user's email" do
+    sign_in user
+    visit profiles_me_path
+
+    expect(page).to have_text(user.email)
+  end
+
+  it "lists the user's description" do
+    sign_in user
+    visit profiles_me_path
+
+    expect(page).to have_text(user.description)
+  end
+
+  it "shows profiles of other users" do
+    sign_in user
+    visit profile_path(id: user2.id)
+    expect(page).to have_text(user2.full_name)
+  end
```

Comment:
In accordance with a review comment before I would suggest adding a test

- Going to `/profiles/me` redirects to profile URL with own profile ID

Reply:
We're thinking of the redirect as being an implementation detail. I think it's fine to assert on the outcome (what the page shows), not on how the browser gets there (redirect).

Reply:
:+1: